clients/tv/src/services/CommandEmitter.js

```diff
@@ -0,0 +1,17 @@
+import mitt from 'mitt'
+import SpeechService from './SpeechService';
+
+const CommandEmitter = {
+    emitter: mitt(),
+    init() {
+        this.observers = [
+            new SpeechService(this.emitter),
+        ];
+    },
+    emit(command, ...options) {
+        this.emitter.emit(command, { options });
+    }
+}
+CommandEmitter.init();
```

Comment:
Skoro ten init jest wykonywany tylko tutaj, to zamiast dawać go jako property na obiekcie, zróbmy tutaj IIFE